RNTester/js/LayoutContextExample.js

```diff
@@ -0,0 +1,112 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ * @flow
+ */
+
+'use strict';
+
+const React = require('react');
+const {
+  LayoutContext,
+  StyleSheet,
+  View,
+  StatusBar,
+  Button,
+  Platform,
+} = require('react-native');
+
+const BORDER_WIDTH = 2;
+
+type Props = {
+  onExampleExit: () => void,
+};
+
+type State = {
+  hidden: boolean,
+  translucent: boolean,
+};
+
+class LayoutContextExample extends React.Component<Props, State> {
```

Comment:
What do you think about also dumping the content of the context to the screen with JSON.stringify? That can be really handy for seeing what's available and debugging in examples like this.

Reply:
👍